gui/src/components/modelSelection/platform/AssistantSelect.tsx

```diff
@@ -297,13 +299,15 @@ export default function AssistantSelect() {
               <span>Assistants</span>
               <div
                 className="flex cursor-pointer flex-row items-center gap-1 hover:brightness-125"
-                onClick={(e) => {
+                onClick={async (e) => {
                   e.stopPropagation();
-                  refreshProfiles();
+                  setLoading(true)
+                  await refreshProfiles()
+                  setLoading(false)
                   buttonRef.current?.click();
                 }}
               >
-                <ArrowPathIcon className="text-lightgray h-2.5 w-2.5" />
+                <ArrowPathIcon className={cn("text-lightgray h-2.5 w-2.5", loading && 'animate-spin')} />
```

Comment:
```suggestion
                <ArrowPathIcon className={cn("text-lightgray h-2.5 w-2.5", loading && 'spin-slow')} />
```

https://github.com/continuedev/continue/blob/main/gui/tailwind.config.cjs#L26